json-endpoints/class.wpcom-json-api-comment-endpoint.php

```diff
@@ -127,7 +127,7 @@ function get_comment( $comment_id, $context ) {
 				);
 				break;
 			case 'author' :
-				$response[$key] = (object) $this->get_author( $comment, 'edit' === $context && current_user_can( 'edit_comment', $comment->comment_ID ) );
+				$response[$key] = (object) $this->get_author( $comment, in_array( $context, array( 'edit', 'display' ) ) && current_user_can( 'edit_comment', $comment->comment_ID ) );
```

Comment:
I think it's safe to drop it since the [`switch` statement above](https://github.com/Automattic/jetpack/blob/master/json-endpoints/class.wpcom-json-api-comment-endpoint.php#L61) would default to `WP_Error` if one of these two values is not provided.